synapseAnnotations/sage_controlled_vocabulary_schema.csv

```diff
@@ -542,7 +550,7 @@ Il1rapKO,"triple mutant strain carries a humanized ApoE knock-in mutation, a 387
 Kif21bT82TSNP,"triple mutant strain carrying a T82T point mutation of the Kif21b gene, a humanized ApoE knock-in mutation, and an R47H point mutation of Trem2",,,,FALSE,modelSystemName,,https://www.jax.org/strain/031938
 MAPT-P301K,,,,,FALSE,modelSystemName,,
 Meox2KOHET,"triple mutant strain carrying a humanized ApoE knock-in mutation, a Cre recombinase knock-in/knock-out allele of the Meox2 gene, and an R47H point mutation of Trem2",,,,FALSE,modelSystemName,,https://www.jax.org/strain/033770
-MthfrC677TSNP,triple mutant strain carrying a humanized ApoE knock-in mutation; an R47H point mutation of Trem2; and a mutant of the Mthfr gene carrying a point mutation that models the human C677T polymorphism,,,,FALSE,modelSystemName,,https://www.jax.org/strain/030922
+MthfrC677TSNP,triple mutant strain carrying a humanized ApoE knock-in mutation, an R47H point mutation of Trem2, and a mutant of the Mthfr gene carrying a point mutation that models the human C677T polymorphism,,,,FALSE,modelSystemName
```

Comment:
This is interesting. I didn't change the semi-colons, but apparently my app did.